NitroxServer/GameLogic/Bases/BaseManager.cs

```diff
@@ -78,7 +126,7 @@ public void BasePieceConstructionCompleted(NitroxId id, NitroxId baseId)
                         // a fully constructed piece.  Therefor, we always update this attribute to make sure it
                         // is the latest.
                         basePiece.BaseId = baseId;
-                        basePiece.ParentId = Optional.OfNullable(baseId);
+                        basePiece.ParentId = Optional<NitroxId>.OfNullable(baseId);
```

Comment:
Optional<Type> is no longer required with the new implementation of Optional class